common/config/rush/pnpm-lock.yaml

```diff
@@ -122,8 +124,8 @@ packages:
   /@azure-rest/core-client-paging/1.0.0-beta.1:
     dependencies:
       '@azure-rest/core-client': 1.0.0-beta.6
-      '@azure/core-paging': 1.2.0
-      '@azure/core-rest-pipeline': 1.3.0
+      '@azure/core-paging': 1.1.3
+      '@azure/core-rest-pipeline': 1.2.0
```

Comment:
I see unrelated changes here, could you please revert those?

Reply:
Look again